### PR TITLE
KAA-1147: Add C++ page

### PR DESCRIPTION
**Pull request by RostakaGmfun**
/cc @forGGe @rasendubi

### CODE REVIEW

File: doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/index.md [hunk 1]
```diff
@@ -5,4 +5,138 @@ permalink: /:path/
 nav: /:path/Programming-guide/Using-Kaa-endpoint-SDKs/Using-Kaa-Cpp-endpoint-SDK
 sort_idx: 20
 ---
-Using Cpp Kaa endpoint SDK
+
+{% assign root_url = page.url | split: '/'%}
+ +{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
```
Comment (rasendubi) [inline doc/Programming-guide/Using-Kaa-endpoint-SDKs/C++/index.md]: `+` seems to be a result of copy-paste.

**Comment by rasendubi** [comment]
Minor. Otherwise, looks good

**Comment by RostakaGmfun** [comment]
@forGGe ?

**Comment by forGGe** [comment]
Looks good

**Pull request merged**